pkg/trustengine/trustengine.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/SAP/jenkins-library/pkg/log"
 )
 
+// const RefTypeFile = "trustengineSecretFile"
+const RefTypeSecret = "trustengineSecret"
```

Comment:
I would rather see this as part of the config package and only keep the trust parts in here

```suggestion
// const RefTypeTrustengineSecretFile = "trustengineSecretFile"
const RefTypeTrustengineSecret = "trustengineSecret"
```

Reply:
Makes sense, done.